edx_proctoring/static/proctoring/spec/proctored_exam_onboarding_spec.js

```diff
@@ -0,0 +1,227 @@
+        '<a aria-label="Next">' +
+        '<span aria-hidden="true">&raquo;</span>' +
+        '</a>' +
+        '</li>' +
+        '<% } else { %>' +
+        '<li>' +
+        '<a class="target-link" href="#" aria-label="Next" data-page-number="<%= currentPage + 1 %>"' +
+        '>' +
+        '<span aria-hidden="true">&raquo;</span>' +
+        '</a>' +
+        '</li>' +
+        '<% }%>' +
+        '</ul>' +
+        '<div class="clearfix"></div>' +
+        '</div>' +
+        '<div>' +
+        '<ul class="status-checkboxes">' +
+        '<% _.each(onboardingStatuses, function(status){ %>' +
+        '<li>' +
+        '<input type="checkbox" id="<%= status %>" value="<%= status %>" ' +
+        '<% if (filters.includes(status)) { %>checked="true"<% } %>>' +
+        '<label for="<%= status %>">' +
+        '<%- interpolate(gettext(" %(onboardingStatus)s "), ' +
+        '{ onboardingStatus: getOnboardingStatus(status) }, true) %>' +
+        '</label>' +
+        '</li>' +
+        '<% }); %>' +
+        '</ul>' +
+        '</div>' +
+        '<table class="onboarding-status-table">' +
+        '<thead>' +
+        '<tr class="onboarding-status-headings">' +
+        '<th class="username-heading"><%- gettext("Username") %></th>' +
+        '<th class="onboarding-status-heading"><%- gettext("Onboarding Status") %></th>' +
+        '<th class="last-updated-heading"><%- gettext("Last Modified") %> </th>' +
+        '</tr>' +
+        '</thead>' +
+        '<% if (isOnboardingItems) { %>' +
+        '<tbody>' +
+        '<% _.each(onboardingItems, function(item){ %>' +
+        '<tr class="onboarding-items">' +
+        '<td>' +
+        '<%- interpolate(gettext(" %(username)s "), { username: item.username }, true) %>' +
+        '</td>' +
+        '<td>' +
+        '<%- interpolate(gettext(" %(onboardingStatus)s "), ' +
+        '{ onboardingStatus: getOnboardingStatus(item.status) }, true) %>' +
+        '</td>' +
+        '<td><%= getDateFormat(item.modified) %></td>' +
+        '</tr>' +
+        '<% }); %>' +
+        '</tbody>' +
+        '<% } %>' +
+        '</table>' +
+        '<% if (!isOnboardingItems) { %>' +
+        '<p>No onboarding data found.</p>' +
+        '<% } %>' +
+        '</div>' +
+        '</div>';
+        this.server = sinon.fakeServer.create();
+        this.server.autoRespond = true;
+        setFixtures('<div class="student-onboarding-status-container" data-course-id="test_course_id"></div>');
+
+        // load the underscore template response before calling the onboarding status view.
+        this.server.respondWith('GET', '/static/proctoring/templates/student-onboarding-status.underscore',
+            [
+                200,
+                {'Content-Type': 'text/html'},
+                html
+            ]
+        );
+    });
+
+    afterEach(function() {
+        this.server.restore();
+    });
+
+    it('should render the proctored exam onboarding view properly', function() {
```

Comment:
Can you add a test case for backend API returns no data? So the case where there are no enrollments to a course yet.